QuEra backend (#73)
CQCL/pytket-braket: docs/intro.txt

=============

``pytket-braket`` is an extension to ``pytket`` that allows ``pytket`` circuits to

> be executed on quantum devices and simulators via Amazon's Braket service, but it does not currently support the QuEra backend.

Reviewer: ```suggestion
be executed on gate-model quantum devices and simulators via Amazon's Braket service.
```
